azurelinuxagent/common/conf.py

```diff
@@ -170,6 +170,7 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
     "Extensions.InitialGoalStatePeriod": 6,
     "Extensions.GoalStateHistoryCleanupPeriod": 1800,
     "OS.EnableFirewallPeriod": 30,
+    "OS.FirewallRulesLogPeriod": 86400,
```

Comment:
This should be a `Debug` flag as we dont want to introduce this flag to the external customers (primarily because it offers them no benefit)